app/views/arclight/repositories/index.html.erb

```diff
@@ -1,4 +1,5 @@
 <div class="al-repositories">
+	<% @page_title = t('arclight.views.repositories.title') %>
```

Comment:
It looks like the indenting is off a bit here

Reply:
Thanks Lea Ann, sublime text tabs issue. FIXED.